openlibrary/i18n/es/messages.po

```diff
@@ -3611,7 +3611,7 @@ msgstr "Patrocinar libro electrónico"
 
 #: openlibrary/macros/LoanStatus.html:91
 #, python-format
-msgid "We don’t have this book yet. You can add it to our Lending Library with a %(price)s tax deductible donation."
+msgid "We don't have this book yet. You can add it to our Lending Library with a %(price)s tax deductible donation."
```

Comment:
This is a workaround, not a fix. UTF-8 should be legal in msgids.

Reply:
:+1: Agree; but that's something that can be fixed in a future pr